vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -313,11 +313,6 @@
       "source": "parts/linux/cloud-init/artifacts/sshd_config_1604",
       "destination": "/home/packer/sshd_config_1604"
```

Comment:
can we remove 1604 as well since we no longer build it?

maybe something for a separate PR

Reply:
sure, I can do that in a different PR!